pkg/mcs/resourcemanager/server/config.go

```diff
@@ -99,6 +101,9 @@ type ControllerConfig struct {
 	// LTBMaxWaitDuration is the max wait time duration for local token bucket.
 	LTBMaxWaitDuration typeutil.Duration `toml:"ltb-max-wait-duration" json:"ltb-max-wait-duration"`
 
+	// LTBTokenRPCMaxDelay is the upper bound of backoff delay for local token bucket RPC.
+	LTBTokenRPCMaxDelay typeutil.Duration `toml:"ltb-token-rpc-max-delay" json:"ltb-token-rpc-max-delay"`
```

Comment:
LTB is not a general abbreviation, Is it more appropriate to use the full name? for example: LocalTokenBucketMaxBackoffDelay. rest LGTM

Reply:
It is the same as the previous name, best to maintain consistency.